docs/maintainers/mapping.rst

```diff
@@ -0,0 +1,27 @@
+MBID Mapping
+============
+
+For a background on how the mapping works, see :ref:`developers-mapping`
```

Comment:
Probably add that the source of the data in the tables is the json dump database on aretha and that a daily cronjob in listenbrainz-mbid-mapping container (not the listenbrainz-mbid-mapping-writer-prod) recreates it.